tests/test_validate_outputs.py

```diff
@@ -0,0 +1,492 @@
+        self.check_pipetasks(
+            ["writeForcedSourceTable",
+             "writeForcedSourceOnDiaObjectTable"],
+            self._num_exposures,
+            self._num_exposures
+        )
+        self.check_pipetasks(
+            ["transformForcedSourceTable",
+             "transformForcedSourceOnDiaObjectTable",
+             "consolidateForcedSourceTable",
+             "consolidateForcedSourceOnDiaObjectTable"],
+            1,
+            1
+        )
+        self.check_sources(["forced_diff_diaObject"], self._num_exposures_good_templates, self._min_sources)
+        # There are fewer forced sources
+        self.check_sources(["forced_src_diaObject"], self._num_exposures, self._min_sources/2)
+        self.check_datasets(["forced_diff_diaObject_schema", "forced_src_diaObject_schema"], 1)
+
+    def test_skymap(self):
+        """Test existence of skymap."""
+        self.check_datasets(["skyMap"], 1)
+
+    def test_skycorr(self):
+        """Test existence of skycorr."""
+        self.check_pipetasks(["skyCorr"], self._num_visits, self._num_visits)
+        self.check_datasets(["skyCorr"], self._num_exposures)
+
+    def test_faro(self):
+        """Test faro outputs."""
+        self.check_pipetasks(["nsrcMeasVisit", "TE3", "TE4"], self._num_visits, self._num_visits)
+        self.check_datasets(
+            ["metricvalue_info_nsrcMeasVisit",
+             "metricvalue_validate_drp_TE3",
+             "metricvalue_validate_drp_TE4"],
+            self._num_visits
+        )
+
+    def check_aperture_corrections(self, catalog, **kwargs):
+        for alg in ("base_PsfFlux", "base_GaussianFlux"):
+            self.assertTrueDescr(f"{alg}_apCorr in schema", f"{alg}_apCorr" in catalog.schema)
+            self.assertTrueDescr(f"{alg}_apCorrErr in schema", f"{alg}_apCorrErr" in catalog.schema)
+            self.assertTrueDescr(f"{alg}_flag_apCorr in schema", f"{alg}_flag_apCorr" in catalog.schema)
+
+    def check_psf_stars_and_flags(self, catalog, min_stellar_fraction=0.95, do_check_flags=True, **kwargs):
+        primary = catalog["detect_isPrimary"]
+
+        psf_stars_used = catalog["calib_psf_used"] & primary
+
+        ext_stars = catalog["base_ClassificationExtendedness_value"] < 0.5
+        self.assertGreaterDescr(
+            f"At least {min_stellar_fraction} of PSF sources are classified as stars",
+            (ext_stars & psf_stars_used).sum(),
+            min_stellar_fraction*psf_stars_used.sum()
+        )
+
+        if do_check_flags:
+            psf_stars_reserved = catalog["calib_psf_reserved"]
+            psf_stars_candidate = catalog["calib_psf_candidate"]
+            self.assertGreaterEqualDescr(
+                "Number of candidate PSF stars >= sum of used and reserved stars",
+                psf_stars_candidate.sum(),
+                psf_stars_used.sum() + psf_stars_reserved.sum()
+            )
+
+
+if __name__ == "__main__":
+    lsst.utils.tests.init()
```

Comment:
If you are using this then I think you need to also add the `setup_module` function. On the other hand you aren't writing any files in the tests so the init won't have any effect and I don't see special asserts from utils so there is no reason to use lsst.utils.tests at all.